### PR TITLE
Dynamic drush alias for Silta environments

### PR DESCRIPTION
**Pull request by Rade333**

### CODE REVIEW

File: drush/sites/self.site.yml [hunk 1]
```diff
@@ -11,4 +11,15 @@
...
 #  root: /path/to/drupal
-#  uri: http://stage.example.com
\ No newline at end of file
+#  uri: http://stage.example.com
+#
+# For Silta environments, you can use this wildcard alias.
+# Replace [repo-name] with the name of your repository, and leave out the square brackets.
+# Usage: drush @branch-name status
+#'*':
+#  uri: https://${env-name}.[repo-name].silta.wdr.io
```
Comment (kirkkala) [inline drush/sites/self.site.yml]: Actually the `[repo-name]` does not apply here if silta.yml `projectName` defines the domain. Would it be too confusing here to example as
```
uri: https://${env-name}.[repo-name OR projectName from silta.yml].silta.wdr.io
```

File: drush/sites/self.site.yml [hunk 2]
```diff
@@ -11,4 +11,15 @@
 #  host: stage.domain.com
 #  user: www-admin
 #  root: /path/to/drupal
-#  uri: http://stage.example.com
\ No newline at end of file
+#  uri: http://stage.example.com
+#
+# For Silta environments, you can use this wildcard alias.
+# Replace [repo-name] with the name of your repository, and leave out the square brackets.
+# Usage: drush @branch-name status
```
Comment (kirkkala) [inline drush/sites/self.site.yml]: For usage, would be good to note the possible forward slashes of branch-name `/` need to be replaced with a dash `-`.

**Comment by misterjoonas** [comment]
What is the status of this PR? Do the dynamic aliases work without code changes?

**Comment by Rade333** [comment]
> What is the status of this PR? Do the dynamic aliases work without code changes?

@misterjoonas as this PR is veery old I believe it won't work out of the box. Feel free to try it out though! I unfortunately don't have the time right now to look into it :/

**Comment by kirkkala** [comment]
@misterjoonas , @Rade333 I updated the PR with SiltaAliasAlterCommands (cbd4f08) and should work now.

I would also like to credit whoever came up with this `SiltaAliasAlterCommands` file I've been copying accross projects. @Jancis probably? 🙏

**Comment by Rade333** [comment]
> @misterjoonas , @Rade333 I updated the PR with SiltaAliasAlterCommands ([cbd4f08](https://github.com/wunderio/drupal-project/commit/cbd4f08244c816927acb51840cf54e704689d31a)) and should work now.
> 
> I would also like to credit whoever came up with this `SiltaAliasAlterCommands` file I've been copying accross projects. @Jancis probably? 🙏

Whoaaaaa this looks GOLDEN! Thank you!

**Comment by Jancis** [comment]
Was not made by me, pretty sure it's @artursv in early April of 2020

**Review by misterjoonas — APPROVED** [review]
Works nicely!
I also managed to make it work with `develop` branch easily which we use for testing.

**Comment by tormi** [comment]
Finally ~~got~~ took time to move this forward. Thanks, @Rade333, @kirkkala, @misterjoonas, @artursv, @Jancis, @mitrpaka - well done!!

**Pull request merged**